app/src/main/java/com/a1ishka/bankaccount/presentation/account/AccountEvent.kt

```diff
@@ -0,0 +1,11 @@
+package com.a1ishka.bankaccount.presentation.account
+
+import com.a1ishka.bankaccount.data.entity.AccountEntity
+
+sealed interface AccountEvent {
+    object SaveAccount : AccountEvent
+    data class DeleteAccount(val account: AccountEntity) : AccountEvent
```

Comment:
as clean architecture concepts says we need separate model classes for each layer. Sometimes we don't do this step as our model doesn't need any mapping for UI or domain. But in my point, it's better when for UI we have separate model (your code is not a mistake, just my thoughts).

Reply:
I was wondering if this would be redundant but I’ll perhaps add it